lib/app/ui/seasons/seasons_page.dart

```diff
@@ -71,17 +42,46 @@ class _SeasonsScreenState extends State<SeasonsScreen> {
                   scrollDirection: Axis.vertical,
                   child: Column(
                     children: [
-                      SizedBox(height: screenHeight * 0.20,),
-                      seasonCardsColumn(context)
+                      SizedBox(
+                        height: screenHeight * 0.20,
+                      ),
+                      _seasonCardsColumn()
                     ],
                   ),
                 ),
               ),
             ),
-            ),
-          const BackgroundTop(title: "Seasons"),
+          ),
+          BackgroundTop(title: widget.title),
         ],
       ),
     );
   }
+
+  Widget _seasonCardsColumn() {
+    final seasonViewModel = Provider.of<SeasonViewModel>(context);
+
+    if (seasonViewModel.isSeasonsFetching) {
+      return SizedBox(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: const Center(
+          child: CircularProgressIndicator(),
+        ),
+      );
+    }
+
+    seasonViewModel.orderSeasonsDescByYear();

```

Comment:
Ezzel az egy sorral tudnék még vitatkozni, mert nem a képernyő feladata lenne ezt meghívni (most legalábbis semmiképp), hanem egyből a REST hívás megkapásakor le lehetne rendezni még a `_seasons` beállítása előtt a viewmodel-en


A fenti függvényt csak akkor kéne hívnia a képernyőnek, ha van egy olyan gomb, ami a rendezést állítja például. A jelenlegi működésben ennek nem kell itt lennie feltétlenül